src/Features/Core/Portable/SemanticSearch/AbstractSemanticSearchService.cs

```diff
@@ -207,7 +236,7 @@ private static bool TryGetFindMethod(Assembly queryAssembly, [NotNullWhen(true)]
         {
             try
             {
-                (method, queryKind) = GetFindMethod(program, ref error);
+                (method, queryKind, isAsync) = GetFindMethod(program, ref error);
             }
             catch
```

Comment:
i don't love the naked catch.  Why not FatalError report this?

Reply:
Actually not needed anymore.